chirp/drivers/radtel_rt493.py

```diff
@@ -0,0 +1,1152 @@
+# Copyright 2025 Paul Oberosler <paul@paulober.dev>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import time
+import struct
+import logging
+from chirp import bitwise, chirp_common, directory, errors, memmap, util
+from chirp.settings import (
+    RadioSetting,
+    RadioSettings,
+    RadioSettingGroup,
+    RadioSettingValueBoolean,
+    RadioSettingValueList,
+    RadioSettingValueInteger,
+    RadioSettingValueString
+)
+
+
+LOG = logging.getLogger(__name__)
+
+COMMAND_ACCEPT = b"\x06"
+CHARSET_HEX = "0123456789ABCDEFabcdef"
+
+
+def _clean_buffer(radio):
+    radio.pipe.reset_input_buffer()
+    radio.pipe.reset_output_buffer()
+    time.sleep(0.002)
```

Comment:
You'll need to explain why you think these resets are necessary, especially with the following junk read (which is fine). These sorts of things behave very differently on different platforms (mainly between windows and the other POSIX platforms) and these tend to be problematic.